prog/postgres/postgres_timeline_nexus.rb

```diff
@@ -57,9 +57,7 @@ def before_run
   end
 
   label def wait_leader
-    hop_destroy if postgres_timeline.leader.nil?
```

Comment:
Not sure how important this was, but this made the timeline disappear before we could upgrade the candidate and attach it to the new timeline.